state/errors.go

```diff
@@ -90,28 +90,33 @@ func IsUnverifiableExtensionError(err error) bool {
 	return errors.As(err, &errUnverifiableExtensionError)
 }
 
-// NoValidChildBlockError is a sentinel error when the case where a certain block has
-// no valid child.
-type NoValidChildBlockError struct {
-	error
+// NoChildBlockError is returned where a certain block has no valid child.
+// Since all blocks are validated before being inserted to the state, this is
+// equivalent to have no stored child.
+type NoChildBlockError struct {
+	err error
```

Comment:
```suggestion
	error
```
I think we took the wrong diff here - on master we embedded error directly to avoid redeclaring `Error()`